model_rnn/__main__.py

```diff
@@ -0,0 +1,4 @@
+from .scripts import train
```

Comment:
You can make it as a cli like [in here](https://github.com/pomelofashion/ml-data-science/blob/feature/add-content-to-vec/content-to-vec/content_to_vec/__main__.py).